### DISCUSSION TITLE
Unlock grid sizes on Xiaomi pad 5

### DISCUSSION THREAD

**Discussion by ljransom** [MonwF/customiuizer#32] [Ideas]
Hey, thanks for this great mod, it's been a lifesaver for several features I required. The only thing I'm missing to make this perfect are grid sizes. Currently they are disabled and there's no option in the menu to change them. The current unlock grid sizes option doesn't seem to have any impact to enable this. Is this something you could add? Thank you.

**Comment by MonwF** [comment]
i don't have any resources or time to work on it. sorry
